pom.xml

```diff
@@ -487,76 +520,6 @@
         </plugins>
     </build>
 
-    <dependencyManagement>
```

Comment:
Why did you remove dependecy management?

Reply:
dependencyManagement doesn't make sense in this repo since we have no submodules to manage dependencies for. It just makes the pom more verbose with no real benefit